fvtest/gctest/GCConfigTest.cpp

```diff
@@ -37,6 +37,17 @@
 #define STRINGFY(str) DO_STRINGFY(str)
 #define DO_STRINGFY(str) #str
 
+const char *gcTests[] = {"fvtest/gctest/configuration/sample_GC_config.xml",
```

Comment:
I don't see why this is necessary -- the config files allow tests to be added/excluded using a text editor and without recompiling, which is useful in debugging scenarios. we can live with this but why not just initGoogle() appropriately as per commit comment and google test prescribed usage?